.github/workflows/acceptance.yml

```
@@ -20,8 +20,9 @@ jobs:
       # node cache
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v1
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
```

This can happen separately, but github deprecated this syntax for setting output. Now it should be:
```suggestion
        run: echo "dir=$(yarn config get cacheFolder)" >> $GITHUB_OUTPUT
```

---

I planned to remove them all (just pushed the change) in favour of using the cache support of setup-x actions.